modules/UI/util/JitsiPopover.js

```diff
@@ -76,7 +76,9 @@ var JitsiPopover = (function () {
      */
     JitsiPopover.prototype.createPopover = function () {
         $("body").append(this.template);
-        $(".jitsipopover > .jitsipopover-content").html(this.options.content);
+        let popoverElem = $(".jitsipopover > .jitsipopover-content");
+        popoverElem.html(this.options.content);
+        APP.translation.translateElement(popoverElem);
```

Comment:
Well, the translation needs to happen at that specific place, because just after it is the positioning. Doing it without translations can move the popup outside user screen. And this code is buried too deep. It seems to me it is ok currently.